enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -39,8 +39,8 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
   },
   pivotTable: {
     rowToggle: {
-      textColor: "white",
-      backgroundColor: "text-light",
+      textColor: "text-white",
```

Comment:
I made a comment in `.storybook/preview-head.html` I think `textColor: "white"` and `backgroundColor: "white"` is easier to understand imo. Having 2 of the same colors are just confusing.

We're different from tailwind in the sense that tailwind uses `bg` prefix to denote where that color would go. In our case, the `bg-<color>` variant [doesn't ever have the same color as the normal `<color>` counterpart](https://github.com/metabase/metabase/blob/54d9fb225705f5493e9c9860c9a2b7a837399ecf/frontend/src/metabase/lib/colors/palette.ts#L37-L44) (expect the `white` color).

if you search the use of `color("white")` 99% is for `background-color` and `color`. So I don't get it why we need to have 3 white colors

- `white`
- `text-white`
- `bg-white`

when `white` is also mostly used for both `color` and `background-color` anyway.

Reply:
[To close the loop](https://metaboat.slack.com/archives/C057WD5L0JG/p1717676739175609?thread_ts=1717663992.584869&cid=C057WD5L0JG). Let's just keep `white` and remove `bg-white` and `text-white`. I don't think it's gonna be too hard to use find and replace `bg-white` to `white` and `text-white` to `white`. But please let me know what you think.

Reply:
I think the only time we should be differentiating between `bg-` and `text-` is if we're dealing with variants like `light`, `medium`, and `dark`. If it's a base color we should have one thing. If we change the base color (i.e. we want a dark theme and we set `white` to a different color, they should all have the same transformation

Reply:
Yeah that's right @oisincoveney Here's the [decision from another thread.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1717684958978589?thread_ts=1717677247.911649&cid=C063Q3F1HPF)